buildpack.go

```diff
@@ -240,6 +256,16 @@ func NewBuildpackMetadata(metadata map[string]interface{}) (BuildpackMetadata, e
 				d.PURL = v
 			}
 
+			if v, ok := v["deprecation_date"].(string); ok {
+				deprecationDate, err := time.Parse(time.RFC3339, v)
+
+				if err != nil {
+					return BuildpackMetadata{}, err
```

Comment:
```suggestion
					return BuildpackMetadata{}, fmt.Errorf("unable to parse deprecation date\n%w", err)
```
Small change to wrap error